src/mock_api.rs

```diff
@@ -26,6 +26,12 @@ impl OsmosisMockApi {
     }
 }
 
+impl Default for OsmosisMockApi {
```

Comment:
Should we rename `OsmosisMockApi` to just `MockApi` since it can be used for all chains?

Reply:
Ah I see now that this was created since `MockApi` from `cosmwasm-std` didn't previously support long enough addresses. It seems to have been fixed now so I'll delete the file: https://github.com/CosmWasm/cosmwasm/commit/b7cd07277ade64e073b88d680c30c848da611fa4